operator/README.md

```diff
@@ -136,6 +136,13 @@ It is important to proceed with caution when deleting a Karmada CR due to the po
 kubectl delete karmada karmada-demo -n test
 ```
 
+If you want to delete a Karmada CR without cascading deletion of the associated Karmada instances,
```

Comment:
`instances` ->  `instance`

I think it should be singular here.

Reply:
thanks,done